packages/legacy/src/Mappings/components/AddEditMappingModal.tsx

```diff
@@ -64,7 +64,7 @@ const useMappingFormState = (
   useFormState({
     name: useFormField(
       '',
-      getMappingNameSchema(mappingsQuery, mappingBeingEdited).label('Name').required()
+      getMappingNameSchema(mappingsQuery, mappingBeingEdited).label('Mapping Name').required()
```

Comment:
For future reference, please do minimal changes when fixing the legacy code, this change can be avoided.

Reply:
Without this change, the error message will not include the entity name, i.e. it will be displayed for mapping field as "Name should be..." instead of "Mapping name should be..."
For Plan and namespace it was already set, so it's nicer to do the same for mapping as well.

Reply:
yes, but this is the legacy code, we should not touch it if we don't need to, in this case we don't need to.